modules/core/quick/tests/src/Kernel/QuickFormTestBase.php

```diff
@@ -29,21 +29,21 @@ abstract class QuickFormTestBase extends KernelTestBase {
   /**
    * Asset entity storage.
    *
-   * @var \Drupal\Core\Entity\EntityStorageInterface
+   * @var \Drupal\asset\AssetStorage
    */
   protected $assetStorage;
```

Comment:
Using the more specific entity type storage interface, in combination with `phpstan-drupal` entity mapping, helps PHPStand understand types for all entities that are created/loaded from the entity storage without additional type hints in our code, eg `AssetStorage` -> `AssetInterface`